### DISCUSSION TITLE
Does ghostty it have a find/search feature

### DISCUSSION THREAD

**Discussion by david-vega** [ghostty-org/ghostty#4142] [Q&A] [answered]
In most terminal emulators I've used they have a find feature that highlight the term(s). This functionality is normally bind to Ctrl+f  or Cmd+f. In some cases it also scrolls to it. and if you are tailing a log every time the term appears it also highlights. I was not able to find this functionality in the docs. Does ghostty have it? 

Here's an example on Konsole (KDE default terminal emulator)
![image](https://github.com/user-attachments/assets/7291448e-a94e-415c-9377-bdaf9e6d1858)

**Comment by JakeWharton** [comment, marked as answer]
See https://github.com/ghostty-org/ghostty/issues/189
